### DISCUSSION TITLE
[v4] Stacking variants with direct children variant

### DISCUSSION THREAD

**Discussion by jacksleight** [tailwindlabs/tailwindcss#15346] [Help] [answered]
Wondering if this isn't supported or I'm just doing it wrong.

Should this work?

```
*-not-first:rounded-l-none
```

As a simpler stacked version of this?

```
[&>*:not(:first-child)]:rounded-l-none
```

**Comment by wongjn** [comment, marked as answer]
Only some variants are stackable and `*` is not a stackable variant. However, you could do `*:not-first:rounded-l-none`: https://play.tailwindcss.com/4uVs0z8NEj

**Reply by jacksleight** [reply]
Ah ha. Perfect, thanks!
